### PR TITLE
test

### PR DESCRIPTION
**Pull request by Foch123**

### CODE REVIEW

**Comment by hassount** [comment]
@Foch123 
Is this a mistaken PR ? if not can we have a better description on what is this PR supposed to add?

**Comment by EwoutH** [comment]
Looks like rewritten CI build scripts (that should properly output exit codes when failing).

**Comment by 1480c1** [comment]
That would be nice

**Comment by 1480c1** [comment]
Although a better pr name would be better

File: citest/cpplint_test.cpp [hunk 1]
```diff
@@ -0,0 +1,11 @@
+nclude <iostream>
```
Comment (joelsole) [inline citest/cpplint_test.cpp]: typo

**Comment by joelsole** [comment]
I am going to close this PR, unless there are opinions on leaving it open

**Pull request closed without merging**